source/_data/scssVariables.json

```diff
@@ -10,6 +10,18 @@
     "dark": "rgb(52, 58, 64)",
     "test": "rgb(255, 192, 203)"
   },
+  "scssBreakpoints": {
```

Comment:
Nice. If we update this in the grid component, this will update correct?

Reply:
these values are coming from bootstrap's $grid-breakpoints(). if needed, they would get overridden probably inside of _variables.scss, similar to how we are able to override $theme-colors()